android/engine/src/main/java/org/smartregister/fhircore/engine/di/CoreModule.kt

```diff
@@ -39,7 +40,7 @@ import org.smartregister.fhircore.engine.util.helper.TransformSupportServices
 class CoreModule {
   @Singleton
   @Provides
-  fun provideSyncJob(@ApplicationContext context: Context) = Sync.basicSyncJob(context)
+  fun provideSyncJob(@ApplicationContext context: Context) = Sync.oneTimeSync<FhirSyncWorker>(context)
```

Comment:
Can we use the subclass extending the `FhirSyncWorker` instead?

Reply:
Should be this one [`AppSyncWorker`](https://github.com/opensrp/fhircore/blob/main/android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt)